### PR TITLE
Add example for using "finder" in authenticator config.

### PR DESCRIPTION
**Pull request by ADmad**
Refs cakephp/cakephp#7164

### CODE REVIEW

File: en/controllers/components/authentication.rst [hunk 1]
```diff
@@ -162,6 +162,41 @@ the following keys:
...
+        $this->loadComponent('Auth', [
+            'authenticate' => [
+                'Form' => [
+                    'finder' => 'auth'
+                ]
+            ],
+        ]);
+    }
+
+This will require your ``UsersTable`` to have finder method ``findAuth()`.
```
Comment (markstory) [inline en/controllers/components/authentication.rst]: Missing a backtick.

**Comment by markstory** [comment]
Other than the syntactic error this looks good to me.

**Comment by markstory** [comment]
We should update the migration guide too.

**Comment by ADmad** [comment]
@markstory Migration guide was already updated in #3044

**Comment by markstory** [comment]
@ADmad derp so it was.

**Pull request merged**